src/Lykke.Service.KycReports.AzureRepositories/Reports/ReportsRepository.cs

```diff
@@ -0,0 +1,148 @@
+
+                return jsonRows;
+            }
+
+            return new List<string>();
+        }
+
+        public async Task<IEnumerable<T>> GetRows<T>(IEnumerable<string> rowKeys = null)
+        {
+            var type = typeof (T);
+            string partitionKey = null;
+
+            if (type == typeof (KycReportDailyLeadership))
+                partitionKey = ReportRowEntity.GeneratePartitionKey(KycReportType.KycReportDailyLeadership);
+                
+            if (!string.IsNullOrWhiteSpace(partitionKey))
+                return await GetReportRows<T>(partitionKey, rowKeys);
+
+            return new List<T>();
+        }
+
+        private async Task<IEnumerable<T>> GetReportRows<T>(string partitionKey, IEnumerable<string> rowKeys = null)
+        {
+            IEnumerable<ReportRowEntity> data;
+            if (rowKeys == null)
+                data = await _tableStorage.GetDataAsync(partitionKey);
+            else
+                data = await _tableStorage.GetDataAsync(partitionKey, rowKeys);
+
+            return data.Select(d => JsonConvert.DeserializeObject<T>(d.JsonRow));
+        }
+
+        public async Task<List<string>> GetKycOfficerStatsJsonRows(DateTime from, DateTime to)
+        {
+            var startDate = new DateTime(from.Year, from.Month, from.Day);
+            var endDate = new DateTime(to.Year, to.Month, to.Day);
+
+            var datesArray = Enumerable.Range(0, 1 + endDate.Subtract(startDate).Days)
+                  .Select(offset => startDate.AddDays(offset))
+                  .ToArray();
+
+            var jsonRows = new List<ReportRowEntity>();
+
+            foreach (var reportDay in datesArray)
+            {
+                var partitionKey = KycOfficerStatsReportEntity.GeneratePartitionKey(reportDay);
+                var data = await _tableStorage.GetDataAsync(partitionKey);
+                jsonRows.AddRange(data);
+            }
+            
+            return jsonRows
+                .OrderByDescending(d => d.Timestamp)
+                .Select(d => d.JsonRow)
+                .ToList();
+        }
+
+        public async Task<List<string>> GetKycOfficersPerformanceJsonRows(DateTime from, DateTime to)
+        {
+            var startDate = new DateTime(from.Year, from.Month, from.Day);
+            var endDate = new DateTime(to.Year, to.Month, to.Day);
+
+            var datesArray = Enumerable.Range(0, 1 + endDate.Subtract(startDate).Days)
+                  .Select(offset => startDate.AddDays(offset))
+                  .ToArray();
+
+            var jsonRows = new List<ReportRowEntity>();
+
+            foreach (var reportDay in datesArray)
+            {
+                var partitionKey = KycOfficersPerformanceReportEntity.GeneratePartitionKey(reportDay);
+                var data = await _tableStorage.GetDataAsync(partitionKey);
+                jsonRows.AddRange(data);
+            }
+
+            return jsonRows
+                //.OrderByDescending(d => d.Timestamp)
```

Comment:
Why is the ordering commented out?

Reply:
There isn't any reason in sorting of result data by time stamp.
Commented code removed.